validation/src/test/scala/hmda/validation/rules/lar/validity/V285Spec.scala

```diff
@@ -0,0 +1,47 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V285Spec extends LarEditCheckSpec {
+
+  property("Succeeds when state is valid FIPS code") {
+    forAll(larGen) { lar =>
+      whenever(lar.geography.state != "NA") {
+        V285(lar) mustBe a[Success]
+      }
+    }
+  }
+
+  property("Fails when state is invalid FIPS code") {
+    forAll(larGen, badStateGen) { (lar, state) =>
+      whenever(lar.geography.state != "NA") {
+        val invalidGeography = lar.geography.copy(state = state)
+        val invalidLar = lar.copy(geography = invalidGeography)
+        V285(invalidLar) mustBe a[Failure]
+      }
+    }
+  }
+
+  property("Succeeds when state is NA and MSA/MD is NA") {
+    forAll(larGen) { lar =>
+      val validGeography = lar.geography.copy(state = "NA", msa = "NA")
+      val validLar = lar.copy(geography = validGeography)
+      V285(validLar) mustBe a[Success]
+    }
+  }
+
+  property("Fails when state is NA but MSA/MD is not NA") {
+    forAll(larGen) { lar =>
+      whenever(lar.geography.msa != "NA") {
```

Comment:
@lgdean I've tweaked the configuration of the Property Checks to decrease the likelihood of this happening. In theory it could still happen, but I got tired of rerunning tests and not seeing failures on any tests for the validation project.